### ISSUE TITLE
Correctly parsing filenames when barcode scan fails

### ISSUE THREAD

**Issue by gusqgm** [fractal-analytics-platform/fractal#48]
following the definition of [metadata](https://github.com/fractal-analytics-platform/mwe_fractal/blob/3d77935c29b782c1174bba0df5f93a938a5bc6c5/fractal/wrappers/conversionTaskWrap.py#L63) in conversionTaskWrap.py, we should have the platename as a first part of the filename until hitting the first "_"  character. This is not going to work in some cases, because of the barcode reader we have placed together with our yokogawa microscope.

What can happen here is that, whenever there is an error with the barcode scanning, a dummy prefix is added instead of it, and so instead of barcode "_" , where "_"  here is the one used for acquiring the plate name have, we have a new prefix with "yymmdd_hhmmss"  (date and time of the barcode scan).
long story short, we can have the following two possible file names:

correct barcode: <barcode>_<img_date>_<img_time>_<well_id>_<time_s, site, chl, z_ind>

failed barcode scan: <scan_date>_<scan_time>_<img_date>_<img_time>_<well_id>_<time_s, site, chl, z_ind>

Unfortunately this error can occur from time to time, and since imaging continues in case of a failed barcode scan, there is no way to go around this, so we need fractal to be able to accept both options already.
